src/views/studio/MyUploadsView/AssetsGroupUploadBar/AssetsGroupUploadBar.style.tsx

```diff
@@ -0,0 +1,108 @@
+import styled from '@emotion/styled'
+import { colors, sizes, zIndex, breakpoints, transitions } from '@/shared/theme'
+
+type ProgressbarProps = {
+  progress: number
+}
+
+type DrawerProps = {
+  isActive?: boolean
+  maxHeight?: number
+}
+
+export const Container = styled.div`
+  position: relative;
+  margin-bottom: ${sizes(6)};
+`
+
+export const AssetsGroupBarUploadContainer = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: ${sizes(4)};
+  width: 100%;
+  height: ${sizes(20)};
+  background-color: ${colors.black};
+  cursor: pointer;
+  transition: all ${transitions.timings.regular} ${transitions.easing};
+`
+export const ProgressBar = styled.div<ProgressbarProps>`
+  --progress-bar-color: #b4bbff33;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: ${({ progress }) => progress && `${progress}%`};
```

Comment:
is it possible to use `transform: scaleX()` here? Would be better for performance.